modules/space/rendering/renderableconstellationsbase.cpp

```diff
@@ -137,19 +139,17 @@ std::string RenderableConstellationsBase::constellationFullName(
                                                       const std::string& identifier) const
 {
     if (_namesTranslation.empty() || identifier.empty()) {
-        std::string message = "List of constellations or the given identifier was empty";
-        LWARNINGC("RenderableConstellationsBase", message);
+        LWARNING("List of constellations or the given identifier was empty");
         return "";
     }
 
     if (_namesTranslation.contains(identifier)) {
         return _namesTranslation.at(identifier);
     }
 
-    std::string message = fmt::format(
+    LERROR(fmt::format(
```

Comment:
Does it make sense to `throw` instead?  Seems like this error is not recoverable